LEAF_Request_Portal/templates/reports/LEAF_Inbox.tpl

```diff
@@ -555,8 +555,7 @@
                             });
                         }
                     });
-                });
-                document.querySelector(`#btn_action${hash}_${stepID}_${data.recordID}`).removeAttribute('disabled');
+                })
```

Comment:
Why is this being removed?
If this is re-added, there should be a check that the selector does not return null before calling its method